app/templates/dashboard.html

```diff
@@ -82,7 +90,7 @@ <h2>Class 4</h2>
         <h1>Join A Class</h1>
         <div class="details">
             <h2>John Doe</h2>
-            <p>Unique ID: </p>
+            <p>Unique ID: {{ current_user.knewbie_id }} </p>
```

Comment:
This one was incomplete it's supposed to be an input field (form) to enter the class code so they can join HAHHA, no need the Knewbie ID here, but don't have to edit this part, I can do it.

Reply:
You should create the join class feature then.